fixtures/js-stubs/groupingConfigs.ts

```diff
@@ -0,0 +1,16 @@
+import {EventGroupingConfig} from 'sentry/types';
```

Comment:
I like to `import type` these so we don't end up dragging in half of sentry - probably still gets dragged in somehow, but you know... baby step 😅 Feel free to leave as is, we can autofix these at some point later